schemas/com.hubspot/deal_deletion/jsonschema/1-0-0

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Webhook event for deleting a deal.",
+  "self": {
+    "vendor": "com.hubspot",
+    "name": "deal_deletion",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "objectId": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 2147483647,
+      "description": "The ID of the object that was created/changed/deleted. For contacts this is the vid; for companies, the companyId; and for deals the dealId."
+    },
+    "changeSource": {
+      "type": "string",
+      "maxLength": 128,
+      "description": "The source of this change. Can be any of the change sources that you find on contact property histories."
+    },
+    "eventId": {
```

Comment:
It seems that docs don't mention any of these fields as required. But wondering if it is really the case.

Reply:
Hey @chuwy , these fields all appears in actual payloads I've received from the webhooks I've set up on a Hubspot portal. I think it's safe to expect them in any of the payloads.